docs/manual/manual.adoc

```diff
@@ -901,62 +901,17 @@ NOTE: Please note that for the rest of `oscap` functionality, unless you specify
 Therefore, you do not need to explicitly validate a datastream before
 use.
 
+=== Validating digital signature in SCAP source data stream
 
-=== SCAP Content Signing and Signature Verification
-The `oscap` itself does not do signature verification. It skips over the
-respective elements. This is due to the fact that there are way too many options
-when it comes to keystores and crypto choices. Instead we recommend users to use
-{xmlsec}[xmlsec1] to verify their SCAP content.  Safely evaluating signed
-content (with signature verification) involves the following steps:
+When evaluating a digitally signed SCAP source data stream OpenSCAP validates
+the digital signature of the data stream. The signature validation is performed
+automatically while loading the file. Data streams with invalid signature are
+rejected and OpenSCAP doesn't evaluate the content. OpenSCAP uses
+https://www.aleksey.com/xmlsec/[XML Security Library] with OpenSSL backend to
+validate the digital signature.
```

Comment:
```suggestion
validate the digital signature.

The signature validation only checks that the datastream hasn't been altered since its latest signature. OpenSCAP doesn't address trustworthiness of certificates or public keys that are part of the `KeyInfo` signature element and that are used to verify the signature. You should verify those keys yourself to prevent evaluation of datastreams that have been modified and signed by bad actors.
```